lib/diagnostic.rb

```diff
@@ -5,7 +5,7 @@ class Diagnostic
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    enum.max { |a, b| a.length <=> b.length }
```

Comment:
Correct! You could also use `max_by`, like so:
`enum.max_by { |entry| entry.length }`